maths/trapezoidal_rule.py

```diff
@@ -1,47 +1,103 @@
 """
 Numerical integration or quadrature for a smooth function f with known values at x_i
 
-This method is the classical approach of suming 'Equally Spaced Abscissas'
+This method is the classical approach of summing 'Equally Spaced Abscissas'
 
-method 1:
-"extended trapezoidal rule"
+Method 1:
+"Extended Trapezoidal Rule"
 
 """
 
 
 def method_1(boundary, steps):
```

Comment:
```suggestion
def trapezoidal_rule(boundary, steps):
```
Let's rename the function to something more descriptive. I'm not sure why the code calls it "method 1" when there's only one method in the file.